src/indra/process/cos.py

```
@@ -0,0 +1,230 @@
+
+    # ── Build lat/lon arrays ─────────────────────────────────────────────────
+    lats = ds["latitude"].values
+    lons = ds["longitude"].values
+    lon_grid, lat_grid = np.meshgrid(lons, lats)
+    flat_lats = lat_grid.ravel()
+    flat_lons = lon_grid.ravel()
+
+    # ── IDW per timestep ─────────────────────────────────────────────────────
+    timesteps = ds["time"].values
+    logger.info(
+        "Running IDW: %d timesteps × %d variables × %d zones = %d calls",
```

Ruff (0.15.9)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)